variables.json

```diff
@@ -36,5 +36,5 @@
   "STORE_ALL_CENTRALIZED_LOGS_IN_FILE": true,
   "SYSLOGPORT": "${install_rsyslogd.rsyslog_port[2]}",
   "TRACKINGKEY": "root-DP@@root-distributePolicy@@00",
-  "RUDDER_INVENTORY_VARS": "{}"
+  "RUDDER_INVENTORY_VARS": "{ "initial-policies":"true" }"
```

Comment:
i didn't want to disable the rudder inventory vars in initial policies,, as they may turn out to be useful in the future

Reply:
what about the "INITIAL" variable that already exists in this file ?

Reply:
I don't understand this line, what in the usage of this inventory variable ?

Reply:
the only goal here is to avoir having an error at installation (read the related ticket).
i'm pretty sure we'll want to have content in there in the future, so i didn't want to remove the reading of this content in initial policies